lib/zipkin-tracer/zipkin_tracer_base.rb

```diff
@@ -17,14 +17,17 @@ def initialize(options={})
 
     def with_new_span(trace_id, name)
       span = start_span(trace_id, name)
+      time_now = Time.now
+      span.timestamp = (time_now.to_f * 1_000_000).to_i # to microseconds
       result = yield span
+      span.duration = ((Time.now - time_now) * 1_000_000).to_i # to microseconds
```

Comment:
- `time_now` is not a very good name, it should be for instance `start_time` or something.
- if we are to do this in other tracers it might be good to have some shared method so that everyone uses the same logic. at the minimum there should be some "to_microseconds" method. if possible a method to wrap the whole thing, doing the time math around the block would be cool.